libbeat/docs/shared-command-line.asciidoc

```diff
@@ -39,6 +39,15 @@ Start http server for profiling. This option is useful for troubleshooting and p
 Write memory profile data to the specified output file. This option is useful for
 troubleshooting the Beat.
 
+*`-path.config`*::
```

Comment:
Yes, we need an extra "guide" to explain this things, I'll make that in a separate PR and then put links to that page from the reference sections. This is the [issue](https://github.com/elastic/beats/issues/1371) explaining the paths and this is the [Elasticsearch explanations page](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-dir-layout.html) that I plan to mostly follow for our docs as well.